src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/VSResources.resx

```diff
@@ -171,15 +171,6 @@ In order to debug this project, add an executable project to this solution which
   <data name="ImportsFromTargetCannotBeDeleted" xml:space="preserve">
     <value>{0} - Import comes fom target file. This import cannot be removed.</value>
   </data>
-  <data name="Restore_PropertyWithInconsistentValues" xml:space="preserve">
```

Comment:
These string resources were moved from `src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/VSResources.resx`  to `src/Microsoft.VisualStudio.ProjectSystem.Managed/Resources.resx` because the code that is using this string resource was moved to ProjectSystem.Managed project.